.golangci.yml

```diff
@@ -17,6 +17,8 @@ linters:
 linters-settings:
   exhaustive:
     default-signifies-exhaustive: true
+  goimports:
+    local-prefixes: github.com/smartcontractkit/chainlink
```

Comment:
:shrug: 
> Put imports beginning with prefix after 3rd-party packages.
    It's a comma-separated list of prefixes.
    Default: ""


https://golangci-lint.run/usage/linters/#goimports